### PR TITLE
Add test for new count parameter.

### LINKED ISSUE #42: Add test for HTTP API for new count feature
There is a new count feature in the nodejs API that we need to test at the HTTP and client levels:

https://github.com/digitalbazaar/bedrock-edv-storage/blob/b6cf3013cd2d793caa6bf0a9adfd633acbafc2e4/test/mocha/25-http-edv-client.js#L372-L376

a new parameter `count: true` can be placed alongside `has` to return just a count of the matching document instead of the entire document.

We need at least one additional test that covers this.

### PR DESCRIPTION
**Pull request by JSAssassin**
fixes #42

### CODE REVIEW

**Comment by aljones15** [comment]
thank you for doing this the node 10 test is failing though.

**Comment by mattcollier** [comment]
This requires a dev branch of edv-client which should be specified in the test/package.json file here until we get that released.

**Comment by mattcollier** [comment]
@JSAssassin would it be possible to add another count query that produces a different result?

File: test/mocha/25-http-edv-client.js [hunk 1]
```diff
@@ -393,20 +393,39 @@ describe('bedrock-edv-storage HTTP API - edv-client', () => {
...
+          invocationSigner: capabilityAgent.getSigner(),
+        });
       } catch(e) {
         err = e;
       }
       assertNoError(err);
-      result.should.be.an('object');
-      result.count.should.equal(4);
+      result1.should.be.an('object');
+      result1.count.should.equal(4);
```
Comment (mattcollier) [inline test/mocha/25-http-edv-client.js:424]: there's another useful assertion which is `result1.should.have.keys()` in the chai API if you want to check for exactly a certain set of object properties.  Please apply that to these results.

**Pull request merged**